packages/core/src/types.ts

```diff
@@ -1325,3 +1327,12 @@ export type InterpreterFrom<
 >
   ? Interpreter<TContext, TStateSchema, TEvent, TTypestate>
   : never;
+
+export interface ActorContext {
+  parent?: SpawnedActorRef<any, any>;
+}
+
+export interface Behavior<TEvent extends EventObject, TEmitted = any> {
```

Comment:
Are we changing `TEmitted` to `TSnapshot` in v4? Or is it more of a v5 change?

```suggestion
export interface Behavior<TEvent extends EventObject, TSnapshot = any> {
```

Reply:
Let's do this in v5